aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -61,14 +62,15 @@ Tensor _mps_convolution(
    bias_defined = bias_opt->defined();
 
   auto memory_format = input_t.suggest_memory_format();
+  bool is_channels_last = (memory_format == at::MemoryFormat::ChannelsLast);
   auto output_t = at::empty(
                     conv_output_size(input->sizes(), weight->sizes(),
                                      padding, stride, dilation),
                     input->scalar_type(),
                     c10::nullopt,
                     kMPS,
                     c10::nullopt,
-                    memory_format);
+                    c10::nullopt);
```

Comment:
This means that the layout of the output will not be ChannelsLast anymore?

Reply:
right, so we add a Transpose at [line](https://github.com/pytorch/pytorch/pull/83121/files#diff-2b02a4fc9ee55c0e5e6c3ca50de5211843685b34b7f3fd709710b09504ccca78R166) to convert it to NCHW format.